vapid.go

```diff
@@ -59,6 +59,14 @@ func generateVAPIDHeaderKeys(privateKey []byte) (*ecdsa.PublicKey, *ecdsa.Privat
 	}
 }
 
+func decodeVapidKey(key string) ([]byte, error) {
+	bytes, err := base64.URLEncoding.DecodeString(key)
+	if err == nil {
+		return bytes, nil
```

Comment:
should this return the error as well?

Reply:
by definition err is nil in this case so it wouldn't make a difference